dags/common_tasks.py

```diff
@@ -74,8 +80,22 @@ def copy_table(conn_id:str, table:Tuple[str, str]) -> None:
             sql.Identifier(dst_schema), sql.Identifier(dst_table),
             sql.Identifier(src_schema), sql.Identifier(src_table)
         )
+    comment_query = sql.SQL(
```

Comment:
Are there any comments on columns to consider?

Reply:
In these two DAGs, comments are added to tables only